SPECS/systemd/systemd.spec

```diff
@@ -36,6 +36,15 @@ Patch15:          https://github.com/systemd/systemd/commit/8f6b442a78d0b485f044
 # Furthermore, strict mode DoT is not supported before v243.
 Patch16:          CVE-2018-21029.nopatch
 
+#Portablectl patches for --now --enable and --no-block flags support
+Patch100:          100-portabled-allow-to-detach-an-image-with-a-unit-in-li.patch
```

Comment:
Why do these start with 100 instead of continuing from 16?

Reply:
I've actually done this as well, its very convenient to split patches into groups (0-99 CVEs, 100-199 features, 200-299 extra stuff, etc). This means you can add an extra CVE patch onto the sources when they are as close to upstream as possible, but don't have to re-number all subsequent patches.